runtime/j9vm/jvm.c

```diff
@@ -1189,6 +1189,14 @@ preloadLibraries(void)
 	 * location if not preloaded. */
 	preloadLibrary(J9_HARMONY_PORT_LIBRARY_SHIM_DLL_NAME, TRUE);
 #endif /* J9VM_OPT_HARMONY */
+
+#if (defined(S390) && defined(LINUX))
+	void *zlibDLL = preloadLibrary(J9_ZIP_DLL_NAME, TRUE);
+	if (NULL == zlibDLL) {
+		fprintf(stderr,"libj9zlib29.so failed to load: %s\n", J9_ZIP_DLL_NAME);
```

Comment:
libj9zlib29.so shouldn't be hard coded, use j9zlib or is J9_ZIP_DLL_NAME sufficient?

Reply:
> libj9zlib29.so shouldn't be hard coded, use j9zlib or is J9_ZIP_DLL_NAME sufficient?

Discussed offline with @r30shah , the decision is to go with `J9_ZIP_DLL_NAME`.